src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -214,3 +214,8 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('subnet', help='Name or ID of subnet. If name is supplied, `--vnet-name` must be supplied.')
         c.argument('vnet_name', help='Name of a virtual network.')
         c.argument('ip_address', help='IPv4 address or CIDR range.')
+
+    with self.argument_context('acr check-health') as c:
+        c.argument('registry_name', options_list=['-n', '--name'], help='Name of target registry.', required=False)
```

Comment:
No need to declare again, the default one should just work https://github.com/Azure/azure-cli/blob/6868f1f8d8e4b30391426908dcca75f7188d2e7e/src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py#L45

Reply:
Nice catch, done :)